pages/_document.tsx

```diff
@@ -9,6 +9,10 @@ class MyDocument extends Document {
             href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap"
             rel="stylesheet"
           ></link>
+          <link
```

Comment:
Instead of keeping both fonts, can we instead remove the original font and completely replace it with the new one?

Reply:
Of course we can, I'll fix it now